ui/org.eclipse.pde.ui/META-INF/MANIFEST.MF

```diff
@@ -122,7 +121,8 @@ Require-Bundle:
  org.eclipse.equinox.security;bundle-version="[1.2.200,2.0.0)"
 Eclipse-LazyStart: true
 Import-Package: org.eclipse.jdt.debug.ui.console,
- org.eclipse.ui.internal.genericeditor
+ org.eclipse.ui.internal.genericeditor,
+ org.osgi.service.event;version="[1.4.1,2.0.0)"
```

Comment:
You really should not import at a micro version level. Here `[1.4,2.0)` is proper.

```suggestion
 org.osgi.service.event;version="[1.4,2.0)"
```

Reply:
Thanks, seems PDE could be smarter here, I created:
- https://github.com/eclipse-pde/eclipse.pde/issues/401